### PR TITLE
Add -fopenmp

### PR DESCRIPTION
**Pull request by tripzero**
Using openmp on linux

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/CMakeLists.txt [hunk 1]
```diff
@@ -757,8 +757,6 @@ foreach(__CLDNN_CompilerFlagName IN ITEMS "CMAKE_CXX_FLAGS" "CMAKE_C_FLAGS")
         MultiProcessorCompilation
         DeadCodeEliminate
         ExtensionsEnabled
-        TreatWarnAsErrorEnabled
-        WarnLevel4
```
Comment (elvin-nnov) [inline inference-engine/thirdparty/clDNN/CMakeLists.txt]: What are the reasons of this commit? You are removing valuable options and required declarations

**Comment by tripzero** [comment]
On Sat, Sep 14, 2019 at 12:04 PM Andrey Malyshev <notifications@github.com>
wrote:

> *@elvin-nnov* requested changes on this pull request.
> ------------------------------
>
> In inference-engine/cmake/dependencies.cmake
> <https://github.com/opencv/dldt/pull/245#discussion_r324434835>:
>
> > @@ -71,11 +71,7 @@ if (WIN32)
>              ENVIRONMENT "OMP"
>              VERSION_REGEX ".*_([a-z]*_([a-z0-9]+\\.)*[0-9]+).*")
>  elseif(LINUX)
> -    RESOLVE_DEPENDENCY(OMP
>
> When we are configuring Inference Engine with OpenMP threading library, we
> still need an ability to compile vs intel openmp. Your patch eliminates
> this ability. You can introduce new cmake option to distinguish intel or
> gnu, but it should be two different configurations.
>
Agreed. That would be better.


> ------------------------------
>
> In inference-engine/thirdparty/clDNN/CMakeLists.txt
> <https://github.com/opencv/dldt/pull/245#discussion_r324435005>:
>
> > @@ -757,8 +757,6 @@ foreach(__CLDNN_CompilerFlagName IN ITEMS "CMAKE_CXX_FLAGS" "CMAKE_C_FLAGS")
>          MultiProcessorCompilation
>          DeadCodeEliminate
>          ExtensionsEnabled
> -        TreatWarnAsErrorEnabled
> -        WarnLevel4
>
> What are the reasons of this commit? You are removing valuable options and
> required declarations
>
cldnn doesn't compile with gcc 8 because of several warnings. Unless it's
being frequently tested on multiple compilers (or just the latest gcc), it
isn't very useful for users trying to install from source.

**Comment by tripzero** [comment]
Any comments on the new patch?

**Comment by ilya-lavrenov** [comment]
OpenMP is not a target for OpenVINO. We use TBB

**Pull request closed without merging**